index.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+const foo = require('./lib/foo.js');
+
+console.log(foo.foo());
```

Comment:
There is actually no `foo` method here, https://github.com/hawkular/hawkular-apm-opentracing-javascript/pull/1/files#diff-3b0c91a742cd7440a80427c9b2eec636R23